envs/example/ci-ceph_swift/group_vars/all.yml

```diff
@@ -3,6 +3,16 @@ stack_env: example_ci-ceph_swift
 
 state_path_base: /opt/stack/data
 
+swift_floating_ip: "{{ hostvars[groups['swiftnode'][0]][primary_interface]['ipv4']['address'] }}"
+swift_undercloud_floating_ip: "{{ swift_floating_ip }}"
+swift_fqdn: "swift.{{ fqdn }}"
+
+etc_hosts:
+  - name: "{{ fqdn }}"
+    ip: "{{ undercloud_floating_ip }}"
+  - name: "{{ swift_fqdn }}"
+    ip: "{{ swift_undercloud_floating_ip }}"
```

Comment:
This will override what's in defaults-2.0.yml which may be okay, but what's provided by https://github.blueboxgrid.com/blue-box-cloud/sitecontroller/blob/master/roles/jenkins-slave/files/scripts/generate_logging_extra_vars.sh#L13-L17 will override it :/ -- we really need to come up with a better way to handle etc entries -- I think for logging I"ll need just use `lineinfile`